moveit_core/planning_scene/CMakeLists.txt

```diff
@@ -12,19 +13,18 @@ target_link_libraries(${MOVEIT_LIB_NAME}
   moveit_kinematic_constraints
   moveit_robot_trajectory
   moveit_trajectory_processing
-  ${LIBOCTOMAP_LIBRARIES} ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+  ${LIBOCTOMAP_LIBRARIES} ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
 
 install(TARGETS ${MOVEIT_LIB_NAME}
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
-
-if(CATKIN_ENABLE_TESTING)
-  find_package(moveit_resources REQUIRED)
-  include_directories(${moveit_resources_INCLUDE_DIRS})
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include)
 
-  catkin_add_gtest(test_planning_scene test/test_planning_scene.cpp)
-  target_link_libraries(test_planning_scene ${MOVEIT_LIB_NAME})
-endif()
+#TODO: Fix this so it can be built with colcon
+# if(BUILD_TESTING)
```

Comment:
re-enable tests please

Reply:
Tests failing. Let's try to solve https://github.com/AcutronicRobotics/moveit2/issues/13 first please. We should be able to re-enable all of them afterwards.

Reply:
You all fixed this somewhere in https://github.com/AcutronicRobotics/moveit2/pull/71, is that correct?

Reply:
Yes however they should be submitted here (and we should rebase this code as well). @ahcorde started doing it iteratively while we review the changes.

Reply:
+1 to restoring tests before merge